tstyche.config.json

```diff
@@ -1,15 +1,16 @@
 {
   "testFileMatch": [
-    "**/packages/expect/__typetests__/*.test.ts",
-    "**/packages/expect-utils/__typetests__/*.test.ts",
-    "**/packages/jest/__typetests__/*.test.ts",
-    "**/packages/jest-cli/__typetests__/*.test.ts",
-    "**/packages/jest-expect/__typetests__/*.test.ts",
-    "**/packages/jest-mock/__typetests__/*.test.ts",
-    "**/packages/jest-runner/__typetests__/*.test.ts",
-    "**/packages/jest-snapshot/__typetests__/*.test.ts",
-    "**/packages/jest-types/__typetests__/config.test.ts",
-    "**/packages/jest-types/__typetests__/jest.test.ts",
-    "**/packages/jest-worker/__typetests__/*.test.ts"
+    "packages/expect/__typetests__/*.test.ts",
```

Comment:
Avoiding `**` in the beginning of these patterns makes the whole test run 0.2s shorter. That’s tangible.

Reply:
Can we build up the patterns in a loop? Or does tstyche only support json config?

Reply:
Only json config is supported. The pattern will just one line after migrating all tests. Only some two or three PRs are left and I will clean this up (;

Reply:
Also after migrating it will be possible to remove this config file. All type test files are picked up using the default config. Right now this is not possible, because tests with `tsd` assertions are not passing.